### ISSUE TITLE
Vox ponds - complete redesign of post-apocalyptic dimension

### ISSUE THREAD

**Issue by Firch** [Tslat/Advent-Of-Ascension#2340]
This is not balance suggestion, this is a redesign suggestion (not about details, but about the whole dimension.)
## Introduction
**Vox Ponds** is a dimension that supposed to be post-apocalyptic - but now it feels like just weird toxic swamp.

I'm suggesting to improve it's theme and make it more specific  - not just post-apocalyptic dimension, but dimension that currently survives endless war. Also that will allow to implement one scrapped (I think so?) feature - Arathon dimension, but as a biome for Vox Ponds. I'll explain it later in this suggestion.

### Polluted soil
Polluted grass and soil are nice, so the only thing that I suggesting to change is texture.
![voxsoilgrassrender](https://user-images.githubusercontent.com/47253246/110784080-0b80fd80-828b-11eb-94b8-95cccb82999d.png)
### Vox slate
Polluted stone (name of block) is making no sense since it is a stone, not really a soft material to be polluted. 
So here's my variant of stone in Vox Ponds: Vox Slate.
![voxslaterender](https://user-images.githubusercontent.com/47253246/110786696-40db1a80-828e-11eb-8dd7-50367c495e58.png)
It's texture was inspired by real-life striped slate texture:
![изображение](https://user-images.githubusercontent.com/47253246/110786769-594b3500-828e-11eb-89e8-b023cc4822d4.png)
### Toxic waste
Toxic waste is a cool block, I like how it inflicts with debuffs. But it needs additional features:
It will reduce your movement - similar to soul sand blocks
The tool requirement is shovel - makes more sense?
It will be useful in singleplayer and will be renewable.
![voxwasterender](https://user-images.githubusercontent.com/47253246/110789314-8c42f800-8291-11eb-964f-466cc31af97a.png)
### Metal barrels
In Vox Ponds there will be a special little and common structures - dumps, they will have special barrel blocks, that are similar to vanilla composters - Metal Barrels.
![voxwastebarrelrender](https://user-images.githubusercontent.com/47253246/110791813-78e55c00-8294-11eb-8162-bb2edfb8b46a.png)
They can be used to turn random trash items into Toxic Clot (Item used to craft Toxic Waste Blocks). Toxic Clots are basically opposite to bonemeal items that can be thrown - they also can be used as weapons to poison your enemies. If toxic clot collided with any crop/plant block (even leaves), the block will be destroyed without any drop. So they're useful if you need to remove big amount of plants and trash items.
### Siren block
Ambient block that will create loud sounds if soon there will be event in Vox Ponds. About the event I'll tell later.
### Other blocks
I feel like there's should be kind of mutated mangrove trees. Toxicwood is such a boring and lazy name, what if we renamed it into something like mangradio trees (mangrove + radioactive)?

Vox log is cool, I think it can be used with crawling, but I don't like it because it's... well, log. Something like rusted pipe might work better for it's purpose? (Add gameplay feature: when you trying to go inside it, you automatically crawling into it. This will allow to hide from mobs for a short time.)

Also, vox slate bricks. Yes?

### Asphalt(?) Roads
It would be nice if there's will be endless asphalt roads here (with village-like generation, but 10 times larger) - they will play two functions:
Increase speed of player when he walking on roads - kinda similar to Terraria's asphalt
They will also work as paths - it's easier to travel if you have these small reference points.
Asphalt/??? block that increases speed of player can be crafted by smelting Vox Slate.

### Toxic swamp (current biome)
Just biome that we have currently - plains with ponds/lakes, with addition of common structures that already have been mentioned.

Mobs: I would like to have animal mobs here (no robot-like mobs.)
### Ruined Arathon
Biome full of broken and exploded skyscrapers. However, there's no survivors at all. Addition of this biome will make neat contrast in biomes and will revise old scrapped/mentioned dimension - Arathon. 

Ruined Arathon will work almost like mountain biome - these ruins will be difficult to travel through.
Houses will be made of Vox slate & Vox slate bricks. Of course, between skyscrapers there will be asphalt roads.

### Toxic valley
Desert-like biome, little puddles of water and toxic waste blocks generates here, no grass and soil blocks, only slate.
In this biome there will be siren blocks.

Mobs: Alarmos, Destructors. 

#### Bombardment
A little event for this biome. After siren block's warning, there will be big planes that will drop bombs (bombs aren't destroying blocks). Makes Vox Ponds more chaotic, and you will need to avoid this biome, unless you able to do melee damage to planes. Once plane is destroyed, it will drop weapon that will allow rocket-jumping.

**Comment by DiamondWalker** [comment]
Good idea. Not sure why call the city Arathon though, since it's not related at all.
